pkg/domain/filters/pods.go

```diff
@@ -25,8 +26,26 @@ func GeneratePodFilterFunc(filter string, filterValues []string, r *libpod.Runti
 			if err != nil {
 				return false
 			}
-			for _, id := range ctrIds {
-				return util.StringMatchRegexSlice(id, filterValues)
+			for _, want := range filterValues {
+				isRegex := define.NotHexRegex.MatchString(want)
+				if isRegex {
+					re, err := regexp.Compile(want)
+					if err != nil {
+						// FIXME: how to warn?
```

Comment:
just don't, I mean you can add logrus.Warn() but in the current form it would warn for each pod which could result in many duplicated lines printed. I think it is ok to just don't show a warning.

Reply:
Fair enough. Anything else before I repush?

Reply:
not from my side